_posts/2023-05-15-update-in-steering-committee.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "Changes in the steering committee"
+author: bziliani
+summary: Juan Wajnerman is stepping down from his position, with Beta Ziliani taking his place.
+---
+
+Juan Wajnerman has been involved with Crystal since its very beginning, and has served as one of the three Steering Committee members (the other two being Ary Borenzweig and Brian Cardiff). However, he decided to move on with his personal projects and leave this position. I am honoured to be selected as Juan's replacement.
+
+We can consider ourselves lucky to have shared paths with Juan, a great colleague with fine technical qualities that make him stand out. Our doors will be forever open in case he decides to come back to share his knowledge with the community of the language he helped grow.
```

Comment:
```suggestion
We consider ourselves lucky to have shared paths with Juan, a great colleague with fine technical qualities that make him stand out. Our doors will forever be open in case he decides to return to share his knowledge with the community for the language he helped grow.
```